### PR TITLE
Fav account change detection

### PR DESCRIPTION
**Pull request by ascobie**
Favorite account selection was not updating until after you select another account due to observable. 
Change size of 'download directory' string as it was overflowing the background task. (should change css here to auto ellipsis as well rather than wrapping)

### CODE REVIEW

File: app/components/account/browse/account-list/account-list.component.ts [hunk 1]
```diff
@@ -75,7 +75,11 @@ export class AccountListComponent extends ListBaseComponent implements OnDestroy
         if (this.isAccountFavorite(accountId)) {
             this.accountService.unFavoriteAccount(accountId);
         } else {
-            this.accountService.favoriteAccount(accountId);
+            this.accountService.favoriteAccount(accountId).subscribe({
+                complete: () => {
```
Comment (timotheeguerin) [inline app/components/account/browse/account-list/account-list.component.ts]: you can just do subscribe(() => {

});

here instead as it should emit multiple values

Comment (ascobie) [reply app/components/account/browse/account-list/account-list.component.ts]: cool, thanks.

Comment (ascobie) [reply app/components/account/browse/account-list/account-list.component.ts]: just changed it and now the favorite reverted back to its old behavior ...

Comment (ascobie) [reply app/components/account/browse/account-list/account-list.component.ts]: changed to this: 

            this.accountService.favoriteAccount(accountId).subscribe(() => {
                this.changeDetector.markForCheck();
            });

and its not being called any more. should i use next: () => {} instead?

**Pull request merged**